host/types/bootstrap.d.ts

```diff
@@ -0,0 +1,2 @@
+export {};
```

Comment:
I'm confused why there are so many declarations?

In my very little knowledge of declaration files, I would think there would be basically a single declaration file for each frontend for any exposed modules?

Reply:
yeah - these were generated by `tsc` and it went waaaay overboard.
I've been removing some of the extras in other branches, but I'll untangle that and tidy up this branch properly.

Reply:
Ohh they were autogenerated?

I am not sure we need to 'tidy it up  properly'- I'm happy if we do it incrementally as we move forward

Reply:
ok, sounds good to me.